wazuh/wazuh-dashboard-plugins
# Remove config-handler angularJS service

### Description
The goal of this pull request is to refactor the config-handler.js angularJS service, but since it's no longer being used we will remove it.
 
### Issues Resolved
Closes #6623 

### Evidence
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/9343732/d006ea48-72ad-4b84-b51c-67b82e467103)


### Test
- Check the deleted files are no longer imported of used by any other component.
- Go to Server Management -> Settings
- Click en Edit configuration
- Make changes to the cluster mode, for example
- Save the changes and restar de Wazuh server using `Restart manager-node` button
- Verify the configuration is applied correctly

### Check List
